multimodal/tests/unittests/predictor/test_predictor.py

```diff
@@ -440,149 +440,6 @@ def test_customizing_model_names(
             assert hasattr(predictor._learner._config.model, per_name)
 
 
-def test_model_configs():
```

Comment:
Do we need to add these to something like test_hyperparameters() ?

Reply:
This is to test passing a model config through `config`. The hyperparamters are already tested in other tests.